hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/SerializationUtil.java

```diff
@@ -129,6 +133,10 @@ static RuntimeException handleSerializeException(Object rootObject, Throwable e)
             return exception;
         }
         String clazz = rootObject == null ? "null" : rootObject.getClass().getName();
+        if (rootObject instanceof Throwable throwable) {
+            LOGGER.warning("Failed to serialize '" + clazz + "'. It will be logged here and "
```

Comment:
nit: 
```suggestion
        if (rootObject instanceof Throwable throwable) {
            LOGGER.warning("Failed to serialize throwable '" + clazz + "'. It will be logged here and "
```